### PR TITLE
CI: switch to central static check script

### LINKED ISSUE #90: Switch CI setup script to use centralised checks
Switch `.ci/setup.sh` to use the central static check script in the tests repository.

### PR DESCRIPTION
**Pull request by jodh-intel**
Switch the CI script to use the central script in the tests repository.

Fixes #90.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: .ci/setup.sh [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
-test_repo_dir="${GOPATH}/src/${test_repo}"
-
-# Check the commits in the branch
-checkcommits_dir="${test_repo_dir}/cmd/checkcommits"
-(cd "${checkcommits_dir}" && make)
-checkcommits \
-	--need-fixes \
-	--need-sign-offs \
-	--verbose
+go get -d "$tests_repo" || true
```
Comment (grahamwhaley) [inline .ci/setup.sh]: One day I will really confirm if we can use

```
got get -d -u "$tests_repo"/...
```

and avoid that whole `true` thing...

Comment (jodh-intel) [reply .ci/setup.sh]: I'm pretty sure `go get` is dumb and does `exit 1` even with `-d` if no golang source files are found, so the `|| true` is currently required.

**Comment by jodh-intel** [comment]
Blocked on #106.

**Comment by laijs** [comment]
LGTM

**Comment by grahamwhaley** [comment]
going to nudge the CI... not sure if we have that set up right yet though (/me goes to look)

**Comment by grahamwhaley** [comment]
hmm, @jodh-intel - will you need to rebase to pick up the fixes in #106 ?

**Comment by jodh-intel** [comment]
Yep - branch updated.

**Comment by sameo** [comment]
LGTM

**Pull request merged**